BACK-END/LeafLife/api/models/harvests.py

```diff
@@ -14,6 +14,7 @@ class Harvest(Base):
                                 related_name='harvests')
     quantity_harvested = models.PositiveIntegerField()
     notes = models.TextField(blank=True, null=True)
+    date = models.DateField(auto_now_add=True)
```

Comment:
we dont need this date field because the model inherits created_at and updated_at fields from base class